td.vue/src/i18n/en.js

```diff
@@ -253,7 +253,9 @@ const eng = {
         saveAs: 'Save As',
         saveModel: 'Save Model',
         saveModelAs: 'Save Model As',
-        search: 'Search'
+        search: 'Search',
+        next:'next',
```

Comment:
I agree that the i18n will pick up these strings by default if the do not exist in the other `i18n/*.js` files. Could we have another file with these strings applied to prove the principle, say `i18n/ar.js` ?